src/Altinn.App.Api/Extensions/SwaggerFilterExtensions.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Altinn.App.Api.Extensions;
+
+internal static class SwaggerFilterExtensions
+{
+    /// <summary>
+    /// Adds a filter to the swagger documentation to remove paths that are not used.
+    /// </summary>
+    /// <param name="services"></param>
+    public static void AddSwaggerFilter(this IServiceCollection services)
+    {
+        services.Configure<SwaggerGenOptions>(c =>
+        {
+            c.DocumentFilter<DocumentFilter>();
+        });
+    }
+}
+
+internal class DocumentFilter : IDocumentFilter
+{
+    public void Apply(OpenApiDocument swaggerDoc, DocumentFilterContext context)
```

Comment:
Could we ensure somehow that this filter applies only to swagger docs that we "own"? As in, if developers make their own docs we don't touch them

Reply:
Not sure what would be a sensible restriction, but on second thought I think it makes sense to not throw if these are missing, so then the only action would be to remove our duplicate paths. It seems unlikely that someone would like to have these paths in their custom swagger doc.

I could add something like this, but I can't really se a way to detect if this is ours or theirs, that isn't likely to break. (```"Altinn App Api"``` is currently specified in the apps ``Program.cs``)
```c#
if (swaggerDoc.Info?.Title != "Altinn App Api")
{
    return;
}
```